lib/factori/ecto.ex

```diff
@@ -41,6 +41,8 @@ defmodule Factori.Ecto do
 
   def dump_value(value, %{ecto_type: ecto_type_module})
       when is_atom(ecto_type_module) do
+    Code.ensure_loaded(ecto_type_module)
```

Comment:
Let’s use the "bang" version so the code will crash if the module does not exist

```suggestion
    Code.ensure_loaded!(ecto_type_module)
```

Reply:
This turns out to be too aggressive.